### DISCUSSION TITLE
loading forever

### DISCUSSION THREAD

**Discussion by PositiveK** [Chia-Network/chia-blockchain#7384] [Support]
Latest version
Node synced
Wallet synced
*Plots (loading forever)
*Farm Overview  (loading forever)
*Pool (loading forever)

 tried everything i could think of, no joy.
 any suggestions

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
